src/components/loginform/LoginForm.scss

```diff
@@ -0,0 +1,189 @@
+.login-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background-color: #f2f2f2;
+  }
+  
+  .login-left {
+    flex: 1;
+    padding: 20px;
+    min-height: 100vh;
+    display: flex;
+    background: rgba(255, 255, 255, 0.8) url(../../images/loginform/Backgroundlogin.svg);
+    background-size: cover;
+  }
+  
+  .login-left img {
+    width: 100%;
+  }
+  
+  .login-right {
+    flex: 1;
+    padding: 20px;
+    border-radius: 10px;
+    display: flex;
+  flex-direction: column;
+  align-items: center;
+  }
+  
+  
+  .login-right form {
+    display: flex;
+    flex-direction: column;
+  }
+  
+  .login-right label {
+    width: 151px;
+    height: 24px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    letter-spacing: -0.015em;
+    color: #292929;
+    flex: none;
+    flex-grow: 0;
+    flex-grow: 0;
+  }
+  
+  .login-right input {
+    padding: 10px;
+    border: none;
+    border-radius: 5px;
+    box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.1);
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 10px 12px 12px;
+    gap: 10px;
+    border: 1px solid #B1B5CA;
+    border-radius: 6px;
+    flex: none;
+      flex-grow: 0;
+    order: 0;
+    flex-grow: 0;
+  }
+  
+  .login-buttons {
+    display: flex;
+    flex-direction: column;
+    padding-top: 15px;
+  }
+  
+  .login-buttons button {
+    background: #1A3835;
+    border-radius: 6px;
+    color: #fff;
+    border: none;
+    padding: 10px;
+    cursor: pointer;
+  }
+  
+  .checkbox-container {
+    display: flex;
+    align-items: center;
+    margin: 10px 0;
+  }
+  
+  .checkbox-container input[type="checkbox"] {
+    margin-right: 10px;
+    cursor: pointer;
+  }
+  
+  .login-buttons a {
+    width: 128px;
+    height: -30px;
+    left: 1108px;
+    top: 563px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    text-align: right;
+    letter-spacing: -0.015em;
+    text-decoration-line: underline;
+    color: #1A3835;
+  }
+
+
+
+  .logo-type {
+    
+left: 15.28%;
+right: 65.28%;
+top: 48.44%;
+bottom: 48.41%;
+transform: scale(0.7);
+opacity: 1;
+
+  }
+
+ .placeholder {
+
+
+font-style: normal;
+font-weight: 400;
+font-size: 13.8px;
+line-height: 24px;
+color: #272934;
```

Comment:
Because now we use scss you can make variables file into utils folder and collect all colors. for example:

$account-info-dropdown-border: #929a9d;
and than you can use  like color: $account-info-dropdown-border. 
In future when client say I want color of all buttons to be red instead of blue you have to change only in one spot.

https://sass-lang.com/documentation/variables